modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/auth/SaslAuthenticationStrategy.java

```diff
@@ -53,6 +55,9 @@ public void handle(int channel, ChannelHandlerContext ctx, AmqpConnectionHandler
                     .createSaslServer(connectionHandler.getConfiguration().getHostName(), mechanism.toString());
             byte[] challenge = saslServer.evaluateResponse(response.getBytes());
             if (saslServer.isComplete()) {
+                Subject subject = new Subject();
+                subject.getPrincipals().add(new UsernamePrincipal(saslServer.getAuthorizationID()));
+                connectionHandler.secureBroker(subject);
```

Comment:
Shall we rename this method to `initBoker`?

Reply:
Fixed in 066f712. Renamed to `attachBroker` as it makes more sense.